ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly-4.14-upgrade-from-eus-4.12.yaml

```diff
@@ -0,0 +1,144 @@
+base_images:
+  ansible:
+    name: "4.12"
+    namespace: ocp
+    tag: ansible
+  cli:
+    name: "4.12"
+    namespace: ocp
+    tag: cli
+  dev-scripts:
+    name: test
+    namespace: ocp-kni
+    tag: dev-scripts
+  openstack-installer:
+    name: "4.12"
+    namespace: ocp
+    tag: openstack-installer
+  tests-private-postupg:
+    name: tests-private
+    namespace: ci
+    tag: "4.14"
+  tests-private-preupg:
+    name: tests-private
+    namespace: ci
+    tag: "4.12"
+  tools:
+    name: "4.12"
+    namespace: ocp
+    tag: tools
+  upi-installer:
+    name: "4.12"
+    namespace: ocp
+    tag: upi-installer
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.18
+releases:
+  intermediate:
+    candidate:
+      architecture: amd64
+      product: ocp
+      stream: nightly
+      version: "4.13"
+  latest:
+    release:
+      architecture: amd64
+      channel: candidate
+      version: "4.12"
+  target:
+    candidate:
+      architecture: amd64
+      product: ocp
+      stream: nightly
+      version: "4.14"
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: baremetalds-ipi-ovn-ipv4-fips-p2-f28
+  cron: 28 13 12 * *
+  steps:
+    cluster_profile: equinix-ocp-metal-qe
+    env:
+      DEVSCRIPTS_CONFIG: |
+        FIPS_MODE=true
+        IP_STACK=v4
+        NETWORK_TYPE=OVNKubernetes
+      E2E_RUN_INTERMEDIATE_TAGS: '@4.13 and @baremetal-ipi and @upgrade-sanity'
+      E2E_RUN_LATEST_TAGS: '@4.14 and @baremetal-ipi and @upgrade-sanity'
+      UPGRADE_CHECK_RUN_TAGS: '@upgrade-check and @4.14 and @baremetal-ipi'
+      UPGRADE_PRE_RUN_TAGS: '@upgrade-prepare and @4.14 and @baremetal-ipi'
+    test:
+    - ref: cucushift-installer-check
+    - chain: openshift-upgrade-qe-test-paused
+    workflow: cucushift-installer-rehearse-baremetalds-ipi-ovn-ipv4
+- as: gcp-ipi-workload-identity-rt-fips-f14
```

Comment:
how about f28 to keep consistent with other jobs.

Reply:
Sure, updated.